files/fr/glossary/thread/index.html

```diff
@@ -0,0 +1,23 @@
+---
+title: Fil
+slug: Glossary/Thread
+translation_of: Glossary/Thread
+---
+<p>Dans les sciences de l'informatique, le <strong>fil</strong> correspond au lancement de plusieurs tâches ou programmes au même moment. Chaque unité capable d'exécuter du code est appelée un <strong>fil</strong>.</p>
```

Comment:
I think (at least for the first occurrence) it could be useful to add "d'exécution" between parenthesis (cf. https://fr.wikipedia.org/wiki/Thread_(informatique) ) in order to make it easier for beginner-level readers to distinguish the usual "fil" (de couture) from the concept of threads.

Reply:
Makes sense to me 👍